tests/models/test_cli.py

```diff
@@ -127,48 +128,59 @@ def test_model_with_no_deployable_flavors_fails_pollitely():
         assert "No suitable flavor backend was found for the model." in stderr
 
 
-def test_serve_gunicorn_opts(iris_data, sk_model):
+def test_serve_gunicorn_opts(iris_data, sk_model,
+                             tracking_uri_mock):  # pylint: disable=unused-argument
     if sys.platform == "win32":
         pytest.skip("This test requires gunicorn which is not available on windows.")
     with mlflow.start_run() as active_run:
-        mlflow.sklearn.log_model(sk_model, "model")
-        model_uri = "runs:/{run_id}/model".format(run_id=active_run.info.run_id)
+        mlflow.sklearn.log_model(sk_model, "model", registered_model_name="imlegit")
+        run_id = active_run.info.run_id
+
+    model_uris = [
+        "models:/{name}/{stage}".format(name="imlegit", stage="None"),
+        "runs:/{run_id}/model".format(run_id=run_id)
+    ]
+    for model_uri in model_uris:
```

Comment:
Putting the old logic into a for-loop (diff becomes smaller if you hide whitespace changes e.g. by viewing https://github.com/mlflow/mlflow/pull/2067/files?utf8=%E2%9C%93&diff=unified&w=1)